locale/en.po

```diff
@@ -2279,381 +2279,418 @@ msgstr "Basic"
 msgid "CONFIRM_NORMAL"
 msgstr "Confirm"
 
-#: ../src/general/OptionsMenu.cs:635 ../src/general/OptionsMenu.tscn:295
+#: ../src/general/OptionsMenu.cs:652 ../src/general/OptionsMenu.tscn:298
 msgid "AUTO_RESOLUTION"
 msgstr "Auto ({0}x{1})"
 
-#: ../src/general/OptionsMenu.cs:988
+#: ../src/general/OptionsMenu.cs:662 ../src/general/OptionsMenu.tscn:428
+msgid "AUTO_GPU_NAME"
+msgstr "{0}"
+
+#: ../src/general/OptionsMenu.cs:668 ../src/general/OptionsMenu.cs:674
+#: ../src/general/OptionsMenu.cs:680 ../src/general/OptionsMenu.tscn:453
+msgid "CURRENT_USED_RENDERER_NAME"
+msgstr "{0}"
```

Comment:
I think having many almost exact translation keys will be a bit confusing to the translators and will result in me getting a question about this, so as these values just directly put a technical string here, I'd want to entirely remove these translation keys and just directly assign the final text in to the places used in the GUI. For the scene you can add "PLACEHOLDER" to the editor description and then put an example GPU name. And for the renderer you can just put GLES3 there, it'll work fine as the translation key and text is fine.

Reply:
So, like this?
`// Gles2 is being used
            usedRendererName.Text = "GLES2";`

Reply:
I had decided against putting string literals directly in the codebase without translation as I thought that might have caused some issues with arabic or chinese character related localization. Or did you mean something more like:
`// Gles2 is being used
            usedRendererName.Text = TranslationServer.Translate("GLES2");`?

Reply:
The GLES2 part should be translatable. Just so that one layer of the containerization is removed so `usedRendererName.Text = TranslationServer.Translate("GLES2");`

Reply:
>  Or did you mean something more like:

That's exactly what I meant, and already managed to type a reply before I received an email about your second comment.

Reply:
In the godot editor there's a option named description or sth. Type 'PLACEHOLDER' there.